lms/djangoapps/grades/rest_api/v1/tests/test_grading_policy_view.py

```diff
@@ -122,7 +122,7 @@ def test_not_authenticated(self):
         """
         The view should return HTTP status 401 if user is unauthenticated.
         """
-        self.assert_get_for_course(expected_status_code=401, HTTP_AUTHORIZATION=None)
```

Comment:
@nasthagiri defer this to you.  Is this reasonable?

Reply:
Yup.